pkts-core/src/main/java/io/pkts/framer/FramingException.java

```diff
@@ -0,0 +1,16 @@
+package io.pkts.framer;
+
+import io.pkts.protocol.Protocol;
+
+public class FramingException extends Exception {
+  private final Protocol protocol;
+
+  FramingException(String message, Protocol protocol) {
```

Comment:
nit; final on everything. Will fix that too after merge...

Reply:
Thanks!